Formula/terraform-lsp.rb

```diff
@@ -0,0 +1,46 @@
+class TerraformLsp < Formula
+  desc "Language Server Protocol for Terraform"
+  homepage "https://github.com/juliosueiras/terraform-lsp"
+  url "https://github.com/juliosueiras/terraform-lsp.git",
+      tag:      "v0.0.12",
+      revision: "b0a5e4c435a054577e4c01489c1eef7216de4e45"
+  license "MIT"
+  head "https://github.com/juliosueiras/terraform-lsp.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.Version=#{version}
+      -X main.GitCommit=#{Utils.git_head}
+      -X main.Date=#{time.iso8601}
+    ]
+
+    system "go", "build", *std_go_args(ldflags: ldflags)
+  end
+
+  test do
+    port = free_port
+
+    pid = fork do
+      exec "#{bin}/terraform-lsp serve -tcp -port #{port}"
+    end
+    sleep 2
+
+    begin
+      tcp_socket = TCPSocket.new("localhost", port)
+      tcp_socket.puts <<~EOF
+        Content-Length: 59
+
+        {"jsonrpc":"2.0","method":"initialize","params":{},"id":1}
+      EOF
```

Comment:
Couldn't this be done with cURL?

Reply:
This is the similar same test adopted from the `terraform-ls` formula, let me see if I can use `curl` to do it.

Reply:
I could not get the curl test working

Reply:
I think we can merge the PR and address the test later together with `terraform-ls` one, @SMillerDev what do you think?